test/test_examples_3d_p4est.jl

```diff
@@ -53,8 +53,15 @@ EXAMPLES_DIR = joinpath(pathof(Trixi) |> dirname |> dirname, "examples", "3d")
 
   @trixi_testset "elixir_euler_free_stream_p4est.jl" begin
     @test_trixi_include(joinpath(EXAMPLES_DIR, "elixir_euler_free_stream_p4est.jl"),
-      l2   = [8.116891021528916e-15, 3.5522708607626286e-14, 4.143281237333047e-14, 6.289087484242244e-14, 6.4284229587973e-14],
-      linf = [5.220268661787486e-13, 1.620439893379455e-12, 2.3271384819167906e-12, 2.466915560717098e-12, 5.472955422192172e-12])
+      l2   = [7.839921025049303e-15, 4.3320953097202227e-14, 4.61318381170666e-14, 5.466388734250446e-14, 9.381101254280883e-14],
+      linf = [1.0269562977782698e-12, 4.9040771443742415e-12, 6.0748628349927e-12, 7.261524714863299e-12, 1.1615597372838238e-11],
```

Comment:
What happens if you let this run for a longer time, e.g., `tspan = (0.0, 20.0)` (at least 10,000 time steps)? Do the errors grow further? Given that the initial condition is a **constant**, I find it hard to believe that the error in the linf norm should become this large. Is this normal/expected? It would be great to get your take on this @andrewwinters5000 @fhindenlang!

Reply:
At 20 seconds (16,602 time steps), the errors look like this:
```
────────────────────────────────────────────────────────────────────────────────────────────────────
 Simulation running 'CompressibleEulerEquations3D' with DGSEM(polydeg=4)
────────────────────────────────────────────────────────────────────────────────────────────────────
 #timesteps:              16602                run time:       1.51362420e+03 s
 Δt:             5.13515423e-04                time/DOF/rhs!:  1.94545455e-07 s
 sim. time:      2.00000000e+01
 #DOF:                    89000
 #elements:                 712

 Variable:       rho              rho_v1           rho_v2           rho_v3           rho_e
 L2 error:       5.55108368e-14   1.14710625e-13   1.29052260e-13   2.05167845e-13   1.92276655e-13
 Linf error:     4.17499368e-12   6.94713731e-12   8.67997341e-12   9.90230120e-12   2.41424658e-11
 ∑∂S/∂U ⋅ Uₜ :  -2.29481584e-15
────────────────────────────────────────────────────────────────────────────────────────────────────
```

Reply:
These L2/Linf errors seem reasonable. But to double check can you run the FSP with `tspan=(0.0, 50.0)`

Reply:
Looks nearly identical at `t=50` (41k time steps).
```
────────────────────────────────────────────────────────────────────────────────────────────────────
 Simulation running 'CompressibleEulerEquations3D' with DGSEM(polydeg=4)
────────────────────────────────────────────────────────────────────────────────────────────────────
 #timesteps:              41504                run time:       7.76647962e+02 s
 Δt:             6.81430674e-04                time/DOF/rhs!:  2.76921348e-08 s
 sim. time:      5.00000000e+01
 #DOF:                    89000
 #elements:                 712

 Variable:       rho              rho_v1           rho_v2           rho_v3           rho_e
 L2 error:       5.59147551e-14   1.14594184e-13   1.29060069e-13   2.05090393e-13   1.92203139e-13
 Linf error:     2.78965739e-12   7.04593328e-12   8.38945580e-12   9.17255161e-12   2.43787213e-11
 ∑∂S/∂U ⋅ Uₜ :  -1.65063405e-15
────────────────────────────────────────────────────────────────────────────────────────────────────
```

Reply:
I guess this is OK then.